core/src/testUtils/groovy/nu/yona/server/test/AppService.groovy

```diff
@@ -179,6 +179,12 @@ class AppService extends Service
 		assertUserWithPrivateData(response.responseData, false)
 	}
 
+	def assertUserGetResponseDetailsWithPrivateDataWithUserPhoto(def response)
+	{
+		assertResponseStatusSuccess(response)
+		assertUserWithPrivateData(response.responseData, true)
```

Comment:
The boolean should be false, so the structure validation is not disabled. In that case, the test will fail, as ``PRIVATE_USER_LINKS_NUM_CONFIRMED`` does not include the ``yona:userPhoto`` link.

Reply:
OK, on second thoughts removed this assert method and removed varying links from the default assertion so that they do not disturb the default assertion.